### PR TITLE
#MAR-1805 add training brief builder

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: app/assets/scss/_forms.scss [hunk 1]
```diff
@@ -194,11 +194,19 @@ main {
...
+      display: none;
+    }
+
+    #trainingDetailType .question-optional {
+      display: none;
+    }
+
+    #trainingDetailCover .question-optional {
+      display: none;
     }
```
Comment (tobiasbrown) [inline app/assets/scss/_forms.scss]: Can you use a single selector here?  Like: `.question-optional { display: none; }`

File: app/helpers/buyers_helpers.py [hunk 2]
```diff
@@ -99,3 +99,90 @@ def allowed_email_domain(current_user_id, brief, data_api_client=None):
     email_domain = current_user.get('email_address').split('@')[-1]
     brief_user_email = brief.get('users')[0].get('emailAddress').split('@')[-1]
     return email_domain == brief_user_email
+
+
+def remove_non_cascasde_fields(brief, section, current_question_id, update_data=None):
```
Comment (tobiasbrown) [inline app/helpers/buyers_helpers.py]: Looks like there's a typo in the function name and associated references.

File: app/buyers/views/buyers.py [hunk 3]
```diff
@@ -390,6 +415,25 @@ def update_brief_submission(framework_slug, lot_slug, brief_id, section_id, ques
     )
 
 
+def __navigate_next(content, brief, lot_slug, section_id):
+    if lot_slug == 'training':
+        # if it is the lastone, go to the next section
```
Comment (tobiasbrown) [inline app/buyers/views/buyers.py]: Space needed between `lastone`.

File: app/assets/scss/_forms.scss (hunk 1 above)
Comment (malmas18) [reply app/assets/scss/_forms.scss]: no, because this would hide for all optional questions

File: app/buyers/views/buyers.py (hunk 3 above)
Comment (malmas18) [reply app/buyers/views/buyers.py]: fixed

File: app/helpers/buyers_helpers.py (hunk 2 above)
Comment (malmas18) [reply app/helpers/buyers_helpers.py]: fixed

File: app/helpers/buyers_helpers.py [hunk 4]
```diff
@@ -99,3 +99,90 @@ def allowed_email_domain(current_user_id, brief, data_api_client=None):
     email_domain = current_user.get('email_address').split('@')[-1]
     brief_user_email = brief.get('users')[0].get('emailAddress').split('@')[-1]
     return email_domain == brief_user_email
+
+
+def remove_non_cascade_fields(brief, section, current_question_id, update_data=None):
```
Comment (matt-sm) [inline app/helpers/buyers_helpers.py]: what is this supposed to do?

**Pull request merged**